ci/test.yml

```diff
@@ -324,6 +324,10 @@ storage test:
 hardware legacy device test:
   stage: test
   extends: .legacy_job
+  only:
+    - schedules  # nightly build
+    - /^legacy\/v[0-9.]*$/
```

Comment:
```
/^legacy\/v[0-9\.]*$/
```

or maybe just the following for simplicity:

```
/^legacy\//
```

Reply:
I have simplified it everywhere then. Done in https://github.com/trezor/trezor-firmware/pull/938/commits/4b72c8dff58b30395d00e2b7ac63b8026ba11bef